src/OrbitBase/File.cpp

```diff
@@ -42,14 +46,18 @@ ErrorMessageOr<unique_fd> OpenFileForWriting(const std::filesystem::path& path)
   constexpr int kOpenFlags = O_WRONLY | O_CREAT | O_TRUNC | O_BINARY;
   constexpr int kOpenMode = _S_IREAD | _S_IWRITE;
 #endif  // defined(__linux)
-  int fd = TEMP_FAILURE_RETRY(open(path.string().c_str(), kOpenFlags, kOpenMode));
-
-  if (fd == kInvalidFd) {
-    return ErrorMessage(
-        absl::StrFormat("Unable to open file \"%s\": %s", path.string(), SafeStrerror(errno)));
-  }
+  return OpenFile(path, kOpenFlags, kOpenMode);
+}
 
-  return unique_fd{fd};
+ErrorMessageOr<unique_fd> OpenNewFileForReadWrite(const std::filesystem::path& path) {
```

Comment:
I feel like `OpenFileForWriting` should be renamed to `OpenNewFileForWriting` or this should just be `OpenFileForReadWrite`. The naming is a bit confusing at the moment.

Reply:
well, it does have O_EXCL therefore this is not necesseraly new file